src/sentry/static/sentry/app/components/links/linkWithConfirmation.tsx

```diff
@@ -4,32 +4,48 @@ import classNames from 'classnames';
 
 import Confirm from 'app/components/confirm';
 
+type Props = React.ComponentProps<typeof Confirm> &
+  React.HTMLAttributes<HTMLAnchorElement>;
```

Comment:
Why also anchor?

Reply:
We have an `<a>` element inside `Confirm` and it requires some props specific to anchors.

Reply:
Good suggestions Mr Evan.